scripts/deploy_testing_secret.sh

```diff
@@ -9,10 +9,7 @@ set -o pipefail
 
 REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 
-bins="${REPO_ROOT}/hack/tools/bin"
-k="$bins/kubectl"
-
-cat <<EOF | $k apply -f -
+cat <<EOF | kubectl apply -f -
```

Comment:
Why do we need to pipe an empty stream into `kubectl`?

Reply:
The syntax is ~~a bit~~ very weird but this is saying "cat everything up to the EOF marker (on line 23) to `kubectl apply -f -`".
https://stackoverflow.com/questions/2500436/how-does-cat-eof-work-in-bash

Reply:
I think people sometimes prefer this to the alternative `kubectl apply -f - <<EOF` because it lets you have the input of the pipeline on the left (although you still have the heredoc below it). It makes a bit more sense when there are more commands in the pipeline.

Reply:
Yeah my intent here as well was not to rewrite this script, just fix it to work in a taskfile rather than a Makefile.

Reply:
Aside from cursing the arcane syntax choices of bash, all good here.